docs/catalog.md

```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
   }
   ```
 
+### Using an action sequence and a change trigger to get a document from a Cloudant database
+
+You can use an action sequence in a rule to fetch the document associated with a Cloudant change event.
+
+Create an action that process a document from Cloudant, it will expect a document as a parameter.
+Here is a sample code of an action that handles a document:
+```
+function main(doc){
+  return { "isWalter:" : doc.name === "Walter White"};
+}
+```
+```
+$ wsk action create myAction myAction.js
+```
+
+Create an action sequence that get's the document first, then calls your current `myAction` that expects a document as input
```

Comment:
perhaps prefix by: To read the document from the database, you can use the `read` action in the cloudant package. 

then lead into the sequencing.